### ISSUE TITLE
ScrollingEditBoxTemplate missing setAllPoints on child ScrollBox frame

### ISSUE THREAD

**Issue by Meorawr** [Stanzilla/WoWUIBugs#146]
ScrollingEditBoxTemplate appears to have a defect where its child `ScrollBox` frame is missing a `setAllPoints="true"` attribute. 

If the following demo snippet is loaded, the expectation is that you'd get a nice basic frame with a scrollable editbox inside that contains 100 "Test" lines which can be viewed and scrolled through. In reality, the contents of the scrollable editbox won't show and the inner scrollbox frame will report a zero width and height value.

```lua
local WindowFrame = CreateFrame("Frame", nil, UIParent, "ButtonFrameTemplate")
WindowFrame:SetPoint("CENTER");
WindowFrame:SetSize(640, 480);
ButtonFrameTemplate_HidePortrait(WindowFrame);

local ScrollBar = CreateFrame("EventFrame", nil, WindowFrame, "WowTrimScrollBar");
ScrollBar:SetPoint("TOPRIGHT", -4, -60);
ScrollBar:SetPoint("BOTTOMRIGHT", -4, 28);

local EditFrame = CreateFrame("Frame", nil, WindowFrame, "ScrollingEditBoxTemplate");
EditFrame:SetPoint("TOPLEFT", 5, -60);
EditFrame:SetPoint("BOTTOMRIGHT", ScrollBar, "BOTTOMLEFT");
EditFrame:SetText(string.rep("Test\n", 100));

ScrollUtil.RegisterScrollBoxWithScrollBar(EditFrame:GetScrollBox(), ScrollBar);
```

ScrollingEditBoxTemplate defines the ScrollBox child with only a single anchor point (`TOPLEFT`) and no explicit size. My guess is that there's nothing internal on this template or the scrolling infrastructure that ever attempts to resize or re-anchor the inner ScrollBox, so it ends up always having a (0, 0) size and never actually being able to show its contents.

If we look at another template - ScrollingFontTemplate - one difference is that its own ScrollBox child has the `setAllPoints` attribute applied. Indeed if we run a slightly modified version of the original snippet for this one template, this works fine right away.

```lua
local WindowFrame = CreateFrame("Frame", nil, UIParent, "ButtonFrameTemplate")
WindowFrame:SetPoint("CENTER");
WindowFrame:SetSize(640, 480);
ButtonFrameTemplate_HidePortrait(WindowFrame);

local ScrollBar = CreateFrame("EventFrame", nil, WindowFrame, "WowTrimScrollBar");
ScrollBar:SetPoint("TOPRIGHT", -4, -60);
ScrollBar:SetPoint("BOTTOMRIGHT", -4, 28);

-- Modified to use ScrollingFontTemplate:
local FontFrame = CreateFrame("Frame", nil, WindowFrame, "ScrollingFontTemplate");
FontFrame:SetFontObject(GameFontHighlightSmall);
FontFrame:SetPoint("TOPLEFT", 5, -60);
FontFrame:SetPoint("BOTTOMRIGHT", ScrollBar, "BOTTOMLEFT");
FontFrame:SetText(string.rep("Test\n", 100));

ScrollUtil.RegisterScrollBoxWithScrollBar(FontFrame:GetScrollBox(), ScrollBar);
```

Following this, if we go back to the original snippet it seems that users need to manually invoke the following to actually have a functioning scrollable editbox; so presumably the template should be doing this for us.

```lua
-- Append to the end of the first test snippet to make the scrollable editbox actually work.
EditFrame:GetScrollBox():SetAllPoints(EditFrame);
```

**Comment by Meorawr** [comment]
Should be resolved in a future build.

**Comment by Meorawr** [comment]
Resolved for BCC in the first 2.5.2 PTR build; still affects 9.1 at present.

**Comment by Meorawr** [comment]
This is fixed in 9.1.5. Might have also been fixed during 9.1 already.
